WordPress/Classes/ViewRelated/Stats/Shared Views/StatsTotalRow.swift

```diff
@@ -199,8 +199,9 @@ class StatsTotalRow: UIView, NibLoadable, Accessible {
 
         switch (showDisclosure, hasChildRows) {
         case (true, true):
-            let hint = expanded ? "Expanded. Tap to collapse." : "Collapsed. Tap to expand."
-            accessibilityHint = NSLocalizedString(hint, comment: "Accessibility hint")
+            accessibilityHint = expanded
```

Comment:
Is this related to this PR?

Reply:
Yes, I mentioned it in the description. This is one of the things that was causing `localize.py` to fail. I'll update the PR title to be clear that the issues were not just Xcode 11 related.